tests/test_inference.py

```diff
@@ -1964,6 +1959,20 @@ def test_many_trees(self):
         self.verify(ts)
 
 
+def get_default_inference_sites(sample_data):
+    """
+    Returns the site positions that would be used for inference by
+    default.
+    """
+    inference_sites = []
+    for var in sample_data.variants():
+        counts = tsinfer.allele_counts(var.genotypes)
+        assert len(var.site.alleles) == 2
```

Comment:
I don't get why this is an assert. Surely we allow triallelic etc sites, but just don't adde them to the returned inference sites.

Reply:
It's just a testing function that never gets called on anything more complicated. This seemed better than duplicating the logic of the main function.